### PR TITLE
sci-libs/handyg: new package, add 0.1.5

### PR DESCRIPTION
**Pull request by APN-Pucky**
Signed-off-by: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>

### CODE REVIEW

File: sci-libs/handyg/handyg-0.1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	virtual/fortran
+"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-so.patch
+)
+
+src_configure() {
+	tc-export CC CXX FC AR
+	CC="${CC}" CFLAGS="${CFLAGS} -fPIC" CXX="${CXX}" CXXFLAGS="${CXXFLAGS} -fPIC" FC="${FC}" FFLAGS="${FFLAGS} -fPIC" AR="${AR}" LD="${FC}" ./configure --prefix=/usr LDFLAGS="${LDFLAGS}"
```
[the remark this answers is not in the document]
Comment (APN-Pucky) [reply sci-libs/handyg/handyg-0.1.5.ebuild:31]: Ups, I agree and will check again. They have their own configure script and it takes all these flags as input and env. (LDFLAGS had to be after configure, iirc).

[the remark this answers is not in the document]
Comment (APN-Pucky) [reply sci-libs/handyg/handyg-0.1.5.ebuild]: Without adding `PIC` to FFLAGS it complains about it missing: https://gitlab.com/APN-Pucky/gentoo-hep-forge/-/jobs/3244369046

[the remark this answers is not in the document]
Comment (AndrewAmmerlaan) [reply sci-libs/handyg/handyg-0.1.5.ebuild]: > Without adding `PIC` to FFLAGS it complains about it missing: https://gitlab.com/APN-Pucky/gentoo-hep-forge/-/jobs/3244369046 Replaced version only has that flag.

Strange, 

Let's just keep it like this then. Will test and merge soon.

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1: 

pkgcheck had some minor style complaints, I took the liberty of fixing them while merging:
```
sci-libs/handyg
RedundantLongDescription: metadata.xml longdescription closely matches DESCRIPTION
TrailingEmptyLine: version 0.1.5: ebuild has trailing blank line(s)
UnquotedVariable: version 0.1.5: unquoted variable EPREFIX on line: 30
WhitespaceFound: version 0.1.5: ebuild has leading whitespace on on line: 25
WhitespaceFound: version 0.1.5: ebuild has trailing whitespace on on line: 34
```

**Comment by APN-Pucky** [comment]
Thanks, I just updated my CI this weekend to also show `pkgcheck scan`'s warnings.

**Pull request closed without merging**